### ISSUE TITLE
Support for HCL

### ISSUE THREAD

**Issue by nikolay** [fioncat/otree#29]
It's a popular format within the DevOps domain.

**Comment by fioncat** [comment]
Good idea! I have used HCL before, but not with Rust. So I need to take some times to try and test [hcl-rs](https://docs.rs/hcl-rs/latest/hcl/).

**Comment by fioncat** [comment]
@nikolay Hi, HCL is now available in https://github.com/fioncat/otree/releases/tag/v0.6.1
